src/github.com/getlantern/flashlight/main/pac.go

```diff
@@ -76,31 +76,27 @@ func genPACFile() {
 	}
 	formatter :=
 		`var bypassDomains = %s;
-		function FindProxyForURL(url, host) {
-			if (isPlainHostName(host) // including localhost
-			|| shExpMatch(host, "*.local")) {
+	function FindProxyForURL(url, host) {
```

Comment:
I originally did that, but Microsoft Edge seems very sensitive to the PAC file and my change actually broke that browser.  It turns out that we don't need it, because ui.lantern.io will never get added to the exceptions list because it doesn't go through detour.